Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -1807,6 +1833,16 @@ def slack_send():
         if tags and not any(elem in entry_tags for elem in tags):
             return
 
+        if entry:
```

Comment:
lets call file_entry

Reply:
looks like it can also be a text entry, i will check that it doesn't break any existing mirrored functionality. better not to change its name

Reply:
Checked it out and it mirrors indeed only file entries